### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by xavierlepretre**
This PR adjusts minor stuff before going live.

### R1: Internal review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R2: External review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R3: Internal QA review (B9lab)

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### R4: Final external QA review

- Technical review
  - [ ] requested
  - [ ] completed
- Language review
  - [ ] requested
  - [ ] completed

### RC: Release Candidate

- [ ] Ready to be merged

### CODE REVIEW

**Review by CitMC — APPROVED** [review]

**Review by AndrewB9lab — APPROVED** [review]
Language checks applied

File: ida-customizations/ida-course/quiz-week1.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ You have reached the end of Week 1. Time for a check-up to see how much you have
 
 **Please make sure you enter the same email address used to sign up for this Academy when submitting the quiz.**
 
-Follow this link: [Week 1 Quiz](https://hr.gs/ida-c3-week1-quiz), to submit a pop quiz covering general questions on Cosmos and its main concepts. The quiz contains 37 questions, and you have 120 minutes to complete it.
+Follow this link: [Week 1 Quiz](https://hr.gs/ida-c4-week1-quiz), to submit a pop quiz covering general questions on Cosmos and its main concepts. The quiz contains 37 questions, and you have 120 minutes to complete it. That is a lot more time than we expect you to need.
```
Comment (xavierlepretre) [inline ida-customizations/ida-course/quiz-week1.md:13]: @coldice Is the quiz URL correct?

**Comment by xavierlepretre** [comment]
@coldice This one can go straight in.

File: hands-on-exercise/4-run-in-prod/index.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ tags:
 
 After having developed your chain, now you will take the next step to move your chain to a proof-of-concept and into production.
 
-![](/lp-images/universe.svg)
+![Several planets around a star](/lp-images/universe.svg)
```
Comment (coldice) [inline hands-on-exercise/4-run-in-prod/index.md:16]: thank you for this!

**Review by coldice — APPROVED** [review]

**Pull request merged**